webpack/assets/javascripts/bundle_novnc.js

```diff
@@ -0,0 +1,89 @@
+import RFB from '@novnc/novnc/core/rfb';
+
+let rfb;
+
+function sendCtrlAltDel() {
+  rfb.sendCtrlAltDel();
+  return false;
+}
+
+function showStatus(state, message) {
+  let level;
+  const status = $('#noVNC_status');
+  const ctrlAltDeleteButton = $('#ctrlAltDelButton');
+
+  switch (state) {
```

Comment:
probably make sense to extract to another function or use some sort of CONST?

Reply:
Perhaps a const lookup table might do the trick here; is actually the old code that I reworked just a little bit.